### PR TITLE
Strip querystring (and fragment) from stackframe.file

### PR DESCRIPTION
**Pull request by bengourley**
Querystring and fragments in `stackframe.file` causes a few issues with sourcemaps and grouping. Removing them resolves this.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Bugsnag error reporter for JavaScript
-![10.23kB](https://img.shields.io/badge/size-10.23kB-green.svg)
+![10.29kB](https://img.shields.io/badge/size-10.29kB-green.svg)
```
Comment (bengourley) [inline README.md]: Without 64d618b6380f5338b27c73af07f1dcf5712f0e5e the size became `10.39kB`.

File: browser/plugins/test/strip-query-string.test.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+const plugin = require('../strip-query-string')
+
+const Client = require('../../../base/client')
+const VALID_NOTIFIER = { name: 't', version: '0', url: 'http://' }
+
+describe('plugin: strip query string', () => {
+  it('should strip querystrings from urls', () => {
+    expect(
+      plugin._strip('https://bath.tub.xyz/assets/js/app.js?id=49c3ad475f69072745e3')
```
Comment (kattrali) [inline browser/plugins/test/strip-query-string.test.js]: Will it strip query strings with multiple items?

```js
plugin._strip('https://bath.tub.xyz/assets/js/app.js?id=49c3ad475f69072745e3&name=bob')
```

File: browser/plugins/test/strip-query-string.test.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    ).toBe('https://bath.tub.xyz/assets/js/app.js')
+    expect(
+      plugin._strip('http://bath.tub.xyz/assets/js/app.js#nmfrz')
+    ).toBe('http://bath.tub.xyz/assets/js/app.js')
+    expect(
+      plugin._strip('http://bath.tub.xyz:8808/assets/js/app.js#nmfrz')
```
Comment (kattrali) [inline browser/plugins/test/strip-query-string.test.js]: Will it strip URLs with both query strings and fragments?

File: browser/index.js [hunk 4]
```diff
@@ -3,25 +3,40 @@ const version = '__VERSION__'
...
+const pluginContext = require('./plugins/context')
+const pluginRequest = require('./plugins/request')
+const pluginThrottle = require('../base/plugins/throttle')
+const pluginConsoleBreadcrumbs = require('./plugins/console-breadcrumbs')
+const pluginNavigationBreadcrumbs = require('./plugins/navigation-breadcrumbs')
+const pluginInteractionBreadcrumbs = require('./plugins/interaction-breadcrumbs')
+const pluginInlineScriptContent = require('./plugins/inline-script-content')
+const pluginSessions = require('./plugins/sessions')
+const pluginIp = require('./plugins/ip')
+const pluginStripQueryString = require('./plugins/strip-query-string')
```
Comment (kattrali) [inline browser/index.js]: This seems like a sensible change - plugins had names purely for the sake of fetching them out by name.

File: browser/plugins/test/strip-query-string.test.js (hunk 2 above)
Comment (bengourley) [reply browser/plugins/test/strip-query-string.test.js]: It will but there isn't a test. I'll add one.

File: browser/index.js (hunk 4 above)
Comment (bengourley) [reply browser/index.js]: Yeah, they needed to be accessible by name and also enumerable. But the pretty object keys wouldn't get compressed, so I realised I could just define them all as variables and then stick them in an array for better results.

**Pull request merged**